src/web/components/Core/HighlightedResult/HighlightedResult.spec.tsx

```diff
@@ -2,8 +2,8 @@
 import { render, screen } from '@testing-library/react';
 import Fuse from 'fuse.js';
 
-import { SiteDTO } from '../../../api/services/adminServiceHelpers';
-import { HighlightedResult } from './ParticipantApprovalForm';
+import { SiteDTO } from '../../../../api/services/adminServiceHelpers';
+import { HighlightedResult } from './HighlightedResult';
```

Comment:
not sure why this component was housed in `ParticipantApprovalForm` but moved it to its own file since we are deleting `ParticipantApprovalForm` anyways